[2024-09-11T04:03:55.880+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T04:03:55.921+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v04.get_name scheduled__2022-12-18T00:00:00+00:00 [queued]>
[2024-09-11T04:03:55.943+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v04.get_name scheduled__2022-12-18T00:00:00+00:00 [queued]>
[2024-09-11T04:03:55.944+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-11T04:03:55.971+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2022-12-18 00:00:00+00:00
[2024-09-11T04:03:55.988+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=22097) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T04:03:55.991+0000] {standard_task_runner.py:72} INFO - Started process 22114 to run task
[2024-09-11T04:03:55.989+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v04', 'get_name', 'scheduled__2022-12-18T00:00:00+00:00', '--job-id', '16390', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmp7of727fe']
[2024-09-11T04:03:55.992+0000] {standard_task_runner.py:105} INFO - Job 16390: Subtask get_name
[2024-09-11T04:03:56.101+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v04.get_name scheduled__2022-12-18T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-11T04:03:56.269+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v04' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2022-12-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-12-18T00:00:00+00:00'
[2024-09-11T04:03:56.271+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-11T04:03:56.277+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'fauzi'}
[2024-09-11T04:03:56.328+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-11T04:03:56.329+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v04, task_id=get_name, run_id=scheduled__2022-12-18T00:00:00+00:00, execution_date=20221218T000000, start_date=20240911T040355, end_date=20240911T040356
[2024-09-11T04:03:56.389+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-11T04:03:56.424+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-11T04:03:56.429+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
